addon/webextension/background/senderror.js

```diff
@@ -81,6 +81,16 @@ this.senderror = (function() {
     });
   };
 
+  // Every copy of Screenshots has its own unique moz-extension:// URL.
+  // Raven doesn't correctly group errors with different URLs, so we use the
+  // raven.js `dataCallback` option to turn the UUID into "screenshots-addon"
+  // before sending.
+  // TODO: do this here or somewhere else, though? like in reportError?
+  function normalizeURL(data) {
+    data.request.url = data.request.url.replace(/moz-extension:\/\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/, "resource://screenshots-addon");
```

Comment:
Oh, nice! I had assumed the problem was that Sentry [can't parse](https://github.com/getsentry/raven-js/issues/974) moz-extension URLs, which would explain why it couldn't locate the source. I'll give the raw git approach a try